homework/class_incremets.py

```diff
@@ -0,0 +1,43 @@
+class Car:
+    """
+    This is a car class
+    """
+    nrOfInst = 0
+    def __init__(self, name, color):
+        self.name = name
+        self.color = color
+        Car.nrOfInst += 1
+
+    def show(self):
+        print(f'This is a {self.color} car and  is named {self.name}')
+    @staticmethod
+    def ShowNrOfInst():
+        print(f'The class Car is instanced {Car.nrOfInst}')
+
+class SmallCar(Car):
+        def __init__(self, small_car_name, small_car_color, size):
+            super().__init__(name=small_car_name, color=small_car_color)
+            self.size = size
+
+        def speed(self):
+            return self.size*2
```

Comment:
pep8 recommends to leave spaces between operators.
https://www.python.org/dev/peps/pep-0008/#other-recommendations